src/ILCompiler.Compiler/src/Compiler/DependencyAnalysis/IndirectionNode.cs

```diff
@@ -26,7 +26,7 @@ public void AppendMangledName(NameMangler nameMangler, Utf8StringBuilder sb)
         public int Offset => 0;
 
         public override ObjectNodeSection Section => ObjectNodeSection.DataSection;
-        public override bool IsShareable => false;
+        public override bool IsShareable => true;
```

Comment:
Good point. Fixed and added the dance that lets us use ReadOnly (at least on Windows).